### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by yangshun**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (taneliang) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: /nit Caps?

File: .github/ISSUE_TEMPLATE/Feature_request.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (taneliang) [inline .github/ISSUE_TEMPLATE/Feature_request.md]: E.g. will be more standard I think

File: .github/ISSUE_TEMPLATE/Bug_report.md (hunk 1 above)
Comment (yangshun) [reply .github/ISSUE_TEMPLATE/Bug_report.md]: Actually these are generated by GitHub, I was testing out the feature 😎 

https://github.com/nusmodifications/nusmods/issues/templates/edit

Comment (taneliang) [reply .github/ISSUE_TEMPLATE/Bug_report.md]: That's quite a few typos for a suggested template haha. Is this a new feature? I've never noticed it before

**Comment by yangshun** [comment]
We already have an issue template but on [this page](https://github.com/nusmodifications/nusmods/community) it shows that we don't. I think they changed their detection mechanism. Also, I wanna see how it is like to propose a new issue when the repo has both an issue and a bug report template.

**Comment by yangshun** [comment]
@taneliang Ooo cool, when we create a new issue now we can choose what type of issue it is!

https://github.com/nusmodifications/nusmods/issues/new/choose

**Comment by taneliang** [comment]
Nice!

image

**Pull request merged**